xbmc/platform/android/jni/AudioFormat.cpp

```diff
@@ -34,6 +34,7 @@ int CJNIAudioFormat::ENCODING_DTS_HD                   = 0x00000008;
 int CJNIAudioFormat::ENCODING_DOLBY_TRUEHD             = 0x00000009;
 int CJNIAudioFormat::ENCODING_IEC61937                 = 0x0000000d;
 
+int CJNIAudioFormat::CHANNEL_OUT_MONO                  = 0x00000004;
 int CJNIAudioFormat::CHANNEL_OUT_STEREO                = 0x0000000c;
```

Comment:
I want to change that in the future, this get_static_field helps us to support that on different platforms, but it also makes us incompatible with "real upstream API".

Think of AML's ENCODING_AC3 with the static_field getter we overwrite this field but don't know anymore if we now need to talk RAW or IEC.